projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget-renderer.component.scss

```diff
@@ -1,14 +1,14 @@
-.table-widget-container {
-  display: flex;
-  width: 100%;
-  height: 100%;
+@import 'layout';
+
+.table-content-container {
+  @include flex-layout(column);
+}
 
-  .table {
-    flex: 1 1;
-    width: 100%;
-  }
+.table {
```

Comment:
The previous flex controls were not in a flex container because the content of titled content is nested deeper in the dom. This caused clipping of the pagination controls for the embedded mode (because the table pushed it beyond the container bounds) and hidden data for the full screen mode (because it uses a fixed pagination controls, they were still visible. Instead, the table content went beneath it)